base/_variables.scss

```diff
@@ -1,3 +1,6 @@
+$minFontSize:8;
```

Comment:
Though this is the right way of declaring variable.. I thought if we can do it like
`$fontSizeRange:(
min:8px,
max:50px
)`